src/platyPS/platyPS.psm1

```diff
@@ -637,7 +640,7 @@ function New-ExternalHelp
         }
 
         foreach ($group in $groups) {
-            $maml = GetMamlModelImpl ( $group.Group | ForEach-Object { Get-Content -Raw $_.FullName } )
+            $maml = GetMamlModelImpl ( $group.Group | ForEach-Object { Get-Content -Raw $_.FullName -Encoding $Encode} )
```

Comment:
`Get-Content` doesn't accepts `System.Text.Encoding`. We need to write our own version, similar to what we did with `MySetContent`